Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:16.04
+
+MAINTAINER SHOP Team <ateam@zalora.com>
+
+WORKDIR "/root/"
+
+RUN apt-get update
```

Comment:
usually we have apt-get update && apt-get install to avoid docker cache the update of apt-get